lib/modules/manager/terraform/readme.md

```diff
@@ -1,33 +1,80 @@
-Currently, Terraform supports renovating the following dependencies, where sub-points represent hosting options of the dependencies:
-
-- modules
-  - GitTags
-  - GithubTags
-  - TerraformRegistry ( Public and Private )
-- providers ( deprecated in Terraform 0.13.0 )
-  - TerraformRegistry ( Public and Private )
-- required_providers block ( Terraform >= 0.13.0)
-  - TerraformRegistry ( Public and Private )
-- required_version
-- helm_release
-  - chart repository ( Public and Private )
-- docker\_\*
-  - Docker registry ( Public and Private )
-- kubernetes\_\*
-  - Docker registry ( Public and Private )
-- [tfe_workspace](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/resources/workspace) ( `terraform_version` argument )
```

Comment:
I'm converting this list into a table, as the current layout doesn't render properly on the published docs. The published docs have a "flat list" so you don't even see the sub-points.

I think chopping this up into new headings with a table is easier to read and understand.